extensions/wikia/MercuryApi/MercuryApiController.class.php

```diff
@@ -273,18 +276,15 @@ public function getWikiVariables() {
 
 		$wikiVariables = $this->mercuryApi->getWikiVariables();
 
-		try {
-			$wikiVariables['navData'] = $this->getNavigationData();
-		} catch ( Exception $e ) {
+		$navData = $this->getNavigationData();
+		if ( empty( $navData ) ) {
 			\Wikia\Logger\WikiaLogger::instance()->error(
```

Comment:
Should this be an error? We'll get tickets created by ErrorReporter. Do we know what to do with them?